modules/RTC/DataChannels.js

```diff
@@ -124,19 +123,17 @@ DataChannels.prototype.onDataChannel = function(event) {
                     newValue);
             } else if (colibriClass === 'LastNEndpointsChangeEvent') {
                 // The new/latest list of last-n endpoint IDs.
-
-                self._lastNEndpoints = obj.lastNEndpoints;
+                const lastNEndpoints = obj.lastNEndpoints;
 
                 // The list of endpoint IDs which are entering the list of
                 // last-n at this time i.e. were not in the old list of last-n
                 // endpoint IDs.
                 const endpointsEnteringLastN = obj.endpointsEnteringLastN;
 
-                logger.info(
-                    'Data channel new last-n event: ',
-                        self._lastNEndpoints, endpointsEnteringLastN, obj);
+                logger.info('Data channel new last-n event: ',
+                    lastNEndpoints, endpointsEnteringLastN, obj);
                 self.eventEmitter.emit(RTCEvents.LASTN_ENDPOINT_CHANGED,
```

Comment:
Another idea would be to just add some method to RTC aka "processLastNEndpointChangedMsg" and emit the RTC event from there. You would not have to deal with the listeners this way, but given that most of the events are emitted directly from here it makes sense to leave it this way.